docs/components/servo/_index.md

```diff
@@ -6,18 +6,29 @@ type: "docs"
 description: "A hobby servo is a special type of small motor whose position you can precisely control."
 tags: ["servo", "components"]
 icon: "/components/img/components/servo.svg"
+no_list: true
 # SME: #team-bucket
 ---
 
-The Viam *servo* component supports hobby servos: small motors with built-in closed-loop control.
-Servos can be useful in robotics because their position is easily and precisely controlled.
+{{% alert title="Tip" color="tip" %}}
 
-- The servo component does not support servomotors.
-To configure an industrial servomotor, use the [motor](/components/motor/) component with an [encoder](/components/encoder/).
+The Viam servo component supports [hobby servos](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
```

Comment:
I think keeping the info about `built-in closed-loop control` would be good. And I think this first sentence may go outside, and above the tip. I'm thinking information that is required standard info should probably always be in the normal text rather than in a tip? Might be worth thinking about a bit more though

Reply:
Yeah, I was struggling with phrasing this if including more info about the encoder/position feedback to differentiate servo & encoded motor configuration.

From what I can tell the "built-in closed-loop control" isn't inaccurate but also isn't necessarily the best phrasing if people reading this don't already have a good understanding of open-loop vs closed-loop control systems, and if they do then they're probably more interested in understanding what's happening in that system? @randhid do you have any thoughts on phrasing here?

Reply:
I agree this is more friendly to people who don't understand these concepts or care about them, and if people do then they are likely more interested in our motor APIs. If we expand our servo apis, then this may change but we'll cross that bridge if it happens.